src/pages/index.module.css

```diff
@@ -13,14 +13,22 @@
   --index-gradient-color: var(--ifm-color-primary-lighter);
  }
 
-.heroBanner {
+ html[data-theme='dark'] .heroBanner {
   padding: 8rem 0;
   text-align: center;
   position: relative;
   overflow: hidden;
   background-image: linear-gradient(to bottom, transparent, transparent, var(--index-gradient-color)), url('/img/fastj/fastj_screenshot.png');
 }
 
+html[data-theme='light'] .heroBanner {
```

Comment:
The length of these css segments might warrant having the non-theme content in its own separate sector, leaving only the differences between dark and light in the dark/light segments. Something similar to this:

```css
.heroBanner {
  padding: 8rem 0;
  text-align: center;
  position: relative;
  overflow: hidden;
}

 html[data-theme='dark'] .heroBanner {
  background-image: linear-gradient(to bottom, transparent, transparent, var(--index-gradient-color)), url('/img/fastj/fastj_screenshot.png');
}

html[data-theme='light'] .heroBanner {
  background-image: linear-gradient(to bottom, transparent, transparent, var(--index-gradient-color)), url('/img/fastj/fastj_screenshot_light.png');
}
```

Otherwise, this is fine.

Reply:
Thought of doing that but never did it